source/transfer_list.rst

```diff
@@ -38,6 +38,8 @@ Transfer list requirements
 
 **R3:** The base address of a TE must be the 16-byte aligned address immediately after the end of the previous entry (or TL header, if the TE is the first entry on the TL).
 
+**R4:** All padding bytes between the end of a TE and the start of the next TE (or end of the list) must be set to 0.
```

Comment:
This simplifies the checksum computation, since we can jump over the padding areas.
The tradeoff is the node producer cannot forget to explicitly set the preceding padding area to zero when adding a new entry.
Can this policy be more error prone?

Reply:
Yes I think it is. I don't think this change is a good idea.

Reply:
Removed in latest push.